README.md

```diff
@@ -5,11 +5,25 @@
 Installing
 ---
 
-Requires Python 3.6 or later
+Requires Git and Python 3.6 or later.
 
+#### Clone the repo:
 ```shell
-$ virtualenv venv
+$ git clone https://github.com/NCI-GDC/gdc-maf-tool.git
+```
+#### Create a virtualenv:
+For Linux and Macos:
+```shell
+$ python3 -m venv venv
 $ source venv/bin/activate
+```
+For windows:
+```shell
+$ py -m venv venv
```

Comment:
I didn't know this was an option on Windows. Neat.

Reply:
This is the recommended way to create virtualenv for windows with python3. Got it straight from docs.